### ISSUE TITLE
Import error from vstruct.primitives

### ISSUE THREAD

**Issue by nrvana** [williballenthin/python-pyqt5-vstructui#1]
When I try to pip3 install I am getting the following error:

```
Downloading/unpacking python-pyqt5-vstructui (from -r requirements.txt (line 6))
  Downloading python-pyqt5-vstructui-0.3.0.tar.gz
  Running setup.py (path:/tmp/pip_build_root/python-pyqt5-vstructui/setup.py) egg_info for package python-pyqt5-vstructui
    Traceback (most recent call last):
      File "<string>", line 17, in <module>
      File "/tmp/pip_build_root/python-pyqt5-vstructui/setup.py", line 3, in <module>
        from vstructui import __version__
      File "/tmp/pip_build_root/python-pyqt5-vstructui/vstructui/__init__.py", line 5, in <module>
        from .vstructui import get_parsers
      File "/tmp/pip_build_root/python-pyqt5-vstructui/vstructui/vstructui.py", line 32, in <module>
        from vstruct.primitives import enum_mixin
    ImportError: cannot import name 'enum_mixin'
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):

  File "<string>", line 17, in <module>

  File "/tmp/pip_build_root/python-pyqt5-vstructui/setup.py", line 3, in <module>

    from vstructui import __version__

  File "/tmp/pip_build_root/python-pyqt5-vstructui/vstructui/__init__.py", line 5, in <module>

    from .vstructui import get_parsers

  File "/tmp/pip_build_root/python-pyqt5-vstructui/vstructui/vstructui.py", line 32, in <module>

    from vstruct.primitives import enum_mixin

ImportError: cannot import name 'enum_mixin'
```

I installed vstruct-2.0.1 via pip3 and I only have the following primitives:

```
>>> dir(vstruct.primitives)
['GUID', '__builtins__', '__cached__', '__doc__', '__file__', '__loader__', '__name__', '__package__', '__spec__', 'b2a_hex', 'float_fmts', 'long', 'num_fmts', 'pointersize', 'struct', 'sys', 'v_base', 'v_bitmask', 'v_bytes', 'v_double', 'v_enum', 'v_float', 'v_int16', 'v_int24', 'v_int32', 'v_int64', 'v_int8', 'v_number', 'v_prim', 'v_ptr', 'v_ptr32', 'v_ptr64', 'v_size_t', 'v_snumber', 'v_str', 'v_uint16', 'v_uint24', 'v_uint32', 'v_uint64', 'v_uint8', 'v_wstr', 'v_zstr', 'v_zstr_utf8', 'v_zwstr']
```

What version of vstruct should I be using that has the enum_mixin primitive?

**Comment by nrvana** [comment]
I uninstalled and reinstalled vivisect_vstruct_wb then python-pyqt5-vstructui installed successfully.

This issue can be closed.

**Comment by williballenthin** [comment]
I had _just_ pushed an updated version of `python-pyqt5-vstructui`, so I think you may have benefited from that :-)

Closed in 001568b3af7fad2e59fbfa6d059ec545da70e4ea

**Comment by nrvana** [comment]
Yea, I think so too. Thanks for the update.
